content/index.rst

```diff
@@ -4,173 +4,126 @@ grid_template: true
 image:
   - src: 'pages/faces-hero.jpg'
 headline:
-  - title: 'You have a brilliant idea.'
-    tagline: "We'll make it fly."
-    type: 'Oddly Human Design & Development'
+  - title: 'We help you meet your goals'
+    type: "We don't just write the code…"
 summary: |
-  **Technology is only useful
-  when it solves real problems for real people.**
-  As your partner,
-  we'll help find those opportunities –
-  providing `web application design and development`_,
-  with a human focus from start to finish.
+  **OddBird provides the full range of**
+  `web application strategy, design, and development`_ –
+  integrating with your company to
+  focus every decision around
+  the goals and needs of your users,
+  so that business strategy and user-experience
+  align for long-term success.
 
   .. callmacro:: content.macros.j2#link_button
     :url: '/contact/'
 
-    Get a free consultation
+    Schedule a consultation
 
-  .. _web application design and development: /work/
+  .. _`web application strategy, design, and development`: /work/
 
 
-Software for Humans
-===================
+OddBird
```

Comment:
@mirisuzanne This is only used for the page title. Is `OddBird` better than `Software for Humans | OddBird`?

Reply:
I'd check with @SondraE